Makefile

```diff
@@ -91,9 +91,9 @@ $(GENERATED_BUILD_PROPERTIES_FILE) : VERSION pkg/embedded/templates/version Make
 	# Turn the contents of VERSION file into a properties file value.
 	cat VERSION | sed "s/^/galasactl.version = /1" >> $@ ; echo "" >> $@
 	# Add the `galasa.boot.jar.version` property based on the build.gradle value.
-	cat build.gradle | grep "def galasaBootJarVersion" | cut -f2 -d\' | sed "s/^/galasa.boot.jar.version = /" >> $@
```

Comment:
If you change these values, then you need to also change the pkg/embedded/embedded.go code which reads the contents of this generated file at run-time.

It's mostly used in the `galasactl project create` stuff, which gets called if you run build-locally.sh. So try manually changing the file first and making sure that works ?